_includes/navbar.html

```diff
@@ -23,7 +23,13 @@
                 <li><a href="/getting-started.html" class="localizable-link" data-localize="getting-started-nav-link">Get started</a></li>
                 <li><a href="/history.html" class="localizable-link" data-localize="history-nav-link">History</a></li>
                 <li><a href="/get-involved.html" class="localizable-link" data-localize="get-involved-nav-link">Get Involved!</a></li>
-                <li><a href="/documentation.html" class="localizable-link" data-localize="docs-nav-link">Documentation</a></li>
+                <li class="dropdown">
+                    <a href="#" class="dropdown-toggle localizable-link" data-localize="docs-nav-link" id="dropdownMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Documentation</a>
+                    <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
+                        <a class="dropdown-item" href="docs/en/about-appium/intro" href="#">English</a>
+                        <a class="dropdown-item" href="docs/cn/about-appium/intro">Chinese</a>
```

Comment:
shouldn't language name itself be written in Chinese ? %)

Reply:
yes, good idea :-)